feature/about/src/commonMain/kotlin/com/mifos/feature/about/AboutViewModel.kt

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2024 Mifos Initiative
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * See https://github.com/openMF/android-client/blob/master/LICENSE.md
+ */
+package com.mifos.feature.about
+
+import androidclient.feature.about.generated.resources.feature_about_app_version
+import androidclient.feature.about.generated.resources.feature_about_failed_to_load
+import androidclient.feature.about.generated.resources.feature_about_ic_source_code
+import androidclient.feature.about.generated.resources.feature_about_ic_website
+import androidclient.feature.about.generated.resources.feature_about_icon_twitter
+import androidclient.feature.about.generated.resources.feature_about_license
+import androidclient.feature.about.generated.resources.feature_about_license_sub
+import androidclient.feature.about.generated.resources.feature_about_support_github
+import androidclient.feature.about.generated.resources.feature_about_support_twitter
+import androidclient.feature.about.generated.resources.feature_about_website
+import androidx.compose.ui.graphics.Color.Companion.White
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.launch
+
+class AboutViewModel : ViewModel() {
+
+    private val _aboutUiState = MutableStateFlow<AboutUiState>(AboutUiState.Loading)
+    val aboutUiState = _aboutUiState.asStateFlow()
+
+    fun getAboutOptions() = viewModelScope.launch {
```

Comment:
- creating a viewmodel just to fetch static data and pass it to our composable is extra. we can make a list and pass it to our composables.
- follow the mifos-mobile